akka-http2-support/src/main/scala/akka/http/impl/engine/http2/rendering/HttpResponseHeaderHpackCompression.scala

```diff
@@ -23,46 +24,28 @@ final class HttpResponseHeaderHpackCompression extends GraphStage[FlowShape[Http
   import HttpResponseHeaderHpackCompression._
 
   // FIXME Make configurable
-  final val maxHeaderSize = 4096
   final val maxHeaderTableSize = 4096
-  //  final val useIndexing = false
-  //  final val forceHuffmanOn = false
-  //  final val forceHuffmanOff = false
 
   val in = Inlet[HttpResponse]("HeaderDecompression.in")
   val out = Outlet[Http2SubStream]("HeaderDecompression.out")
 
   override def shape = FlowShape.of(in, out)
 
-  // format: OFF
-  override def createLogic(inheritedAttributes: Attributes) = new GraphStageLogic(shape)
-    with InHandler with OutHandler {
-    // format: ON
-
-    val buf = {
-      val b = ByteBuffer.allocate(4 * maxHeaderSize) // FIXME, just guessed a number here
```

Comment:
I think a reusable `ByteArrayOutputStream` should equally effective but less complex. WDYT?